### PR TITLE
US003 - Entity Placeholder - create new record

### PR DESCRIPTION
**Pull request by Natalyl86**
https://trello.com/c/RLRk8O4h/92-tc03-1js-entity-placeholder-create-new-record

### CODE REVIEW

File: test/specs/entityPlaceholder.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import {userLogin} from "../../helper/methods";
+import selectors from "./../../data/selectors.json";
+import expected from "./../../data/expected.json";
+import testData from "../../data/testData.json";
+
+describe('Placeholder Page', () => {
+    before(() => {
+        browser.maximizeWindow();
+        browser.url('/');
+        //userLogin('tester140@tester.test', 'qG2mJvmXz6');
```
Comment (dashasandler) [inline test/specs/entityPlaceholder.js:11]: It seems that we do not need such a comments here

File: test/specs/entityPlaceholder.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        $(selectors.placeholder.tester100).click();
+        });
+    it('05/PH-Should save the record', () => {
+        $(selectors.placeholder.saveEntityBtn).click();
+        expect($(selectors.placeholder.savedCheckbox).getAttribute("class"))
+            .to.equal(expected.common.checkBoxSq);
+    });
+    });
+
+
```
Comment (Burdishka) [inline test/specs/entityPlaceholder.js:48]:
```suggestion
    });
});

```

File: test/specs/entityPlaceholder.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        $(selectors.placeholder.stringIF).setValue (testData.placeholder.f1String);
+        $(selectors.placeholder.textIF).setValue(testData.placeholder.f2Text);
+        $(selectors.placeholder.intIF).setValue (testData.placeholder.f3Integer);
+        $(selectors.placeholder.decimalIF).setValue(testData.placeholder.f4Decimal);
+        $(selectors.placeholder.dateIF).setValue(testData.placeholder.f5Date);
+        $(selectors.placeholder.dateTimeIF).click()
+        $(selectors.placeholder.userIF).click();
+        $(selectors.placeholder.tester100).scrollIntoView();
+        $(selectors.placeholder.tester100).click();
+        });
```
Comment (Burdishka) [inline test/specs/entityPlaceholder.js:40]:
```suggestion
    });
```

File: test/specs/entityPlaceholder.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    it('04/PH-Should fill in a form', () => {
+        $(selectors.placeholder.stringIF).setValue (testData.placeholder.f1String);
+        $(selectors.placeholder.textIF).setValue(testData.placeholder.f2Text);
+        $(selectors.placeholder.intIF).setValue (testData.placeholder.f3Integer);
+        $(selectors.placeholder.decimalIF).setValue(testData.placeholder.f4Decimal);
+        $(selectors.placeholder.dateIF).setValue(testData.placeholder.f5Date);
+        $(selectors.placeholder.dateTimeIF).click()
+        $(selectors.placeholder.userIF).click();
+        $(selectors.placeholder.tester100).scrollIntoView();
+        $(selectors.placeholder.tester100).click();
```
Comment (Burdishka) [inline test/specs/entityPlaceholder.js:39]: In this "it" you don't have any verification so would be better to move these statements to another "it(5/PH-Should save the record)" with verification

**Pull request merged**